lte/gateway/deploy/roles/magma/files/magma-bridge-reset.sh

```diff
@@ -89,7 +90,10 @@ ovs-vsctl --all destroy Flow_Sample_Collector_Set
 ifdown "$SGI_BR"
 ifdown "$GTP_BR"
 ifdown patch-up
-if [ "$KMOD_RELOAD" != '-y' ] && [ "$KMOD_RELOAD" != '-f' ];
+if [ "$KMOD_RELOAD" != '-n' ];
+then
+  ovs-ofctl del-flows "GTP_BR"
```

Comment:
can you also delete uplink-br flows

Reply:
there is deletion code on line 116
```

# restore OVS flows
ovs-ofctl del-flows "$SGI_BR"
ovs-ofctl add-flows "$SGI_BR" "$FLOW_DUMP"
rm "$FLOW_DUMP"

```

Reply:
ah.. right, lets skip this part for '-n' and jump to flow table reset.